blogposts/2021/better-onboarding-advice-from-3-engineering-leaders.md

```diff
@@ -0,0 +1,126 @@
+---
+title: 'Better onboarding: Here’s how the best engineering leaders do it'
+author: Beyang Liu
+authorUrl: 
+description: 'Building robust software that runs at scale is a team effort. Here are 3 engineering leaders on how they use both human and automated processes to set their growing teams up for success.'
+heroImage: /blog/better-onboarding-advice-from-engineering-leaders.gif
+socialImage: website/static/blog/better-onboarding-advice-from-engineering-leaders.gif
+publishDate: 2021-05-19T18:00-07:00
+tags: [blog]
+slug: better-onboarding-advice-from-engineering-leaders
+published: true
+---
+
+![Better onboarding video game graphic](/blog/better-onboarding-advice-from-engineering-leaders.gif)
+
+_This is the second post in our Better Onboarding series. Check out the first, "[How to prevent codebase overwhelm](https://about.sourcegraph.com/blog/better-onboarding-how-to-prevent-codebase-overwhelm/)"_.
```

Comment:
```suggestion
_This is the second post in our Better Onboarding series. Check out the first, "[How to prevent codebase overwhelm](https://about.sourcegraph.com/blog/better-onboarding-how-to-prevent-codebase-overwhelm/)"_.

```